src/components/calculator/PrevalenceControls.tsx

```diff
@@ -204,9 +216,11 @@ export const PrevalenceControls: React.FunctionComponent<{
     isTopLocation(data.topLocation) &&
     Locations[data.topLocation].subdivisions.length > 1
 
-  const locationSet = isTopLocation(data.topLocation)
+  const locationSet = !data.useManualEntry && isTopLocation(data.topLocation)
 
-  const isManualEntryCurrently = isManualEntry(data.topLocation)
+  const [isManualEntryCurrently, setIsManualEntryCurrently] = useState(
```

Comment:
Nit: I think this is less likely to get accidentally broken in the future if you specify the type of the generic: `useState<boolean>`